infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -294,6 +313,15 @@ public static void startServer(Map<String, Object> opts, boolean bindToAllInterf
 
         // Create a common Server Context for all servers to access.
         try (ServerContext serverContext = new ServerContext(opts)) {
+
+            setupMetrics(serverContext);
+
+            MetricRegistry registry = new MetricRegistry();
+            StatsLogger statsLogger = new CodeHaleLogger("root", registry);
+            PrometheusProvider prometheusProvider = new PrometheusProvider(9999, statsLogger);
+            serverContext.setMetricsProvider(prometheusProvider);
+            prometheusProvider.start();
+
```

Comment:
The above statements seems redundant. What are their difference to the their counterparts inside setupMetrics?